app/config/messaging.js

```diff
@@ -0,0 +1,22 @@
+const connectionDetails = {
+  host: process.env.MESSAGE_QUEUE_HOST,
+  password: process.env.MESSAGE_QUEUE_PASSWORD,
+  username: process.env.MESSAGE_QUEUE_USER
+}
+
+module.exports = {
+  updateEligibilityQueue: {
+    address: process.env.UPDATE_ELIGIBILITY_QUEUE_ADDRESS,
+    type: 'queue',
+    usePodIdentity: process.env.NODE_ENV === 'production',
```

Comment:
Given these are both queues atm, is it worth including `type` and `usePodIdentity` in the `connectionDetails` object and possible changing the name of it to shared config or something less specific to accommodate the extra properties?

Reply:
Good point, will make it so.